docs/installation/working-with-typescript.md

```diff
@@ -0,0 +1,66 @@
+---
+# Scope:
+# * Introduction to TypeScript in CKEditor 5
+# * List and clarify the things that need attention when using TypeScript.
+
+
+category: installation
+order: 29
+modified_at: 2023-03-29
+---
+
+# TypeScript support
+
+CKEditor 5 is built using TypeSript and so has native types definition.
```

Comment:
```suggestion
CKEditor 5 is built using TypeSript. This means it has native type definitions.
```

Reply:
Why wouldn't we accept this proposal? It looks and sounds good.

Reply:
I was extending this section and then forgot to add this correction. I'll add it.